helix-zk/src/main/java/org/apache/helix/zkscale/zk/BasicZkSerializer.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.helix.zkscale.zk;
```

Comment:
one comment  on the path name here. zkscale does not sound a necessary here in the path. The whole point here is we will have a new ZkClient API, not necessarily related to zk scalability, even we claim our API  provides more scalability, we do not have to say that in our module path :). also, "zkscale" itself is confusing, sounds like a weight-scale, or kind of that things.

Reply:
+1 "zkscale" just doesn't sound right..